toolkit/tests/pastaporto/test_tokens.py

```diff
@@ -72,3 +75,19 @@ async def _():
         decode_service_to_service_token(
             test_token, "secret", issuer="test", audience="association-backend"
         )
+
+
+@test("generate a token")
```

Comment:
this should describe what the test is doing

```suggestion
@test("Generate a service to service token")
```